### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by aprilhuang825**

### CODE REVIEW

**Review by Kaz1022 — COMMENTED** [review]
Looks good!

File: routes/favorites.js [hunk 1]
```diff
@@ -6,11 +6,12 @@ module.exports = (db) => {
   // show a user's favorite list
   router.get("/", (req, res) => {
    let queryString = `
-    SELECT favorites.*, items.title, items.price, items.description
+    SELECT DISTINCT favorites.item_id, items.title, items.price, items.description, items.is_available
```
Comment (b1u3too) [inline routes/favorites.js:9]: So we're not displaying the multiple favourites, but our database is storing multiple favourites. It works, but a person could spam the button and fill our database with junk data! Perhaps as an improvement the additional favourite could not be posted if that favourite already exists in the database?

File: views/message-detail.ejs [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```
Comment (b1u3too) [inline views/message-detail.ejs]: This commit is a bugfix for the route, right? Should this file be committed here?

File: views/my-messages.ejs [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (b1u3too) [inline views/my-messages.ejs]: This commit is a bugfix for the route, right? Should this file be committed here?

File: views/new-message.ejs [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```
Comment (b1u3too) [inline views/new-message.ejs]: This commit is a bugfix for the route, right? Should this file be committed here?

**Review by b1u3too — COMMENTED** [review]
Heya April! 
Smart workaround for the multiple display! Do you think there might be a way to address there being multiple favourites in the database if a person mashes that button?  Also noticed the message .ejs template files -- are those the ones we're going with, should they be on this branch with this pull request?

File: views/message-detail.ejs (hunk 2 above)
Comment (aprilhuang825) [reply views/message-detail.ejs]: Yes you're right, the files should not be here, I've removed them

File: routes/favorites.js (hunk 1 above)
Comment (aprilhuang825) [reply routes/favorites.js:9]: I thought about it too but not sure how to implement it

Comment (b1u3too) [reply routes/favorites.js:9]: Yeah... I'm kind of drawing a blank, too! If you'd like a collaborator to be your rubber ducky and pair program with, I'd be happy to take a swing at making it even shinier together. That said, the bug's fixed and the demo's almost ready. I'll go ahead and merge now!

File: views/message-detail.ejs (hunk 2 above)
Comment (b1u3too) [reply views/message-detail.ejs]: Thank you so much! Truly appreciate all your hard work, and hoping comments as a way of talking about our code has been comfortable for you. :)

**Pull request merged**